### ISSUE TITLE
"Unable to add " Tips unfriendly

### ISSUE THREAD

**Issue by fjqingyou** [transmission/transmission#1217]
I can't finish compiling window exe, so I can only give feedback

"Unable to add " Tips unfriendly

current 
```
tr("<p><b>Unable to add \"%1\".</b></p><p>It is a duplicate of \"%2\" which is already added.</p>").
```

but %1 url too long, for example 1000, I Can't get %2, Insufficient screen height,

**Comment by ckerr** [comment]
Agreed, it probably would make sense to ellipsize that string after some point

**Comment by ckerr** [comment]
breadcrumb for anyone wanting to write this:

https://doc.qt.io/archives/qt-4.8/qfontmetricsf.html#elidedText

**Comment by fjqingyou** [comment]
This is not just a Windows platform. Mac platform has the same problem!

**Comment by mikedld** [comment]
See #1410.

**Comment by fjqingyou** [comment]
Sorry, I can't finish compiling windows. I didn't understand the windows compiling tutorial! So I can't test this!

**Comment by mikedld** [comment]
No need in building it yourself, just grab the latest master build from CI which is currently
* x86: https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_WindowsX86/92727?buildTab=artifacts&guest=1
* x64: https://trtc.mikedld.com/buildConfiguration/Transmission_Sanity_WindowsX64/92726?buildTab=artifacts&guest=1

**Comment by fjqingyou** [comment]
transmission-3.00+-r4d7950dcb9-x64.msi  
It doesn't seem complete. After installation. Unable to start. When starting, an error prompt box will pop up. Qt5network is not found

**Comment by mikedld** [comment]
Works just fine for me. Try repairing the installation or removing and re-installing.

**Comment by fjqingyou** [comment]
Thank you. It started normally after re installation.

Now this version, repeatedly submit the same URL, will not get repeated prompts!

**Comment by ckerr** [comment]
Partially fixed by #1410 and then #2709 finishes the job
